legacyCore/src/main/res/layout/notification_dialog.xml

```diff
@@ -10,6 +10,7 @@
         android:id="@+id/title_text"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:text="@string/core_error_no_internet_connection"
```

Comment:
Why is this default?

Reply:
Likely just old and bad code, I removed the defaults in this PR.